director/template/var_kv.go

```diff
@@ -14,31 +14,32 @@ type VarKV struct {
 
 func (a *VarKV) UnmarshalFlag(data string) error {
 	pieces := strings.SplitN(data, "=", 2)
+	name, value := 0, 1
```

Comment:
These should be constants so that it's clear that they're names for previously-magic numbers. [(Rather than something that is reasonable to change by reassigning to later in the code.)]

I would also suggest calling them "name-index" and "value-index" or something similar. Abbreviating "index" as "idx" would probably be okay.

Reply:
good suggestion